src/features/watch.ts

```diff
@@ -12,18 +12,21 @@ export async function watchBuild(
   rebuild: () => void,
   restart: () => void,
 ): Promise<FSWatcher> {
-  const root = process.cwd()
-  if (options.outDir === root) {
-    throw new Error(
-      'options.outDir cannot be an empty string, which will cause the watch to be invalid.',
-    )
-  }
   const { watch } = await import('chokidar')
   const debouncedRebuild = debounce(rebuild, 100)
 
+  const cwd = process.cwd()
+  if (typeof options.watch === 'boolean' && options.outDir === cwd) {
```

Comment:
This judgment is more appropriate to be placed above. If the conditions are met, an error will be thrown directly without importing related packages. 🤔

Reply:
So efficient! 👍